mods/xpanes/init.lua

```diff
@@ -137,7 +137,7 @@ function xpanes.register_pane(name, def)
 		is_ground_content = false,
 		sunlight_propagates = true,
 		description = def.description,
-		tiles = {def.textures[3], def.textures[3], def.textures[1]},
+		tiles = {def.textures[3], def.textures[3], def.textures[2]},
```

Comment:
game_api needs to be updated too

Reply:
https://github.com/minetest/minetest_game/pull/2603/commits/7e9eeac465bd9f1cb756d189722bd0829610899b
I completely forgot about that, thank you (updated_